### PR TITLE
Fix secondary links default value and types

### PR DESCRIPTION
**Pull request by ad-elias**
Fixes broken companies view

### CODE REVIEW

File: packages/twenty-server/src/engine/metadata-modules/field-metadata/dtos/default-value.input.ts [hunk 1]
```diff
@@ -31,7 +31,7 @@ export class FieldMetadataDefaultValueString {
 
 export class FieldMetadataDefaultValueRawJson {
   @ValidateIf((_object, value) => value !== null)
-  @IsObject()
+  @IsObject() // TODO: Should this also allow arrays?
```
Comment (ad-elias) [inline packages/twenty-server/src/engine/metadata-modules/field-metadata/dtos/default-value.input.ts:34]: @FelixMalfait What do you think of this?

Comment (FelixMalfait) [reply packages/twenty-server/src/engine/metadata-modules/field-metadata/dtos/default-value.input.ts:34]: Let's keep it like that imo

File: packages/twenty-server/src/modules/contact-creation-manager/services/create-company.service.ts [hunk 2]
```diff
@@ -157,10 +153,10 @@ export class CreateCompanyService {
     };
   }
 
-  private createCompanyMap(companies: CompanyWorkspaceEntity[]) {
+  private createCompanyMap(companies: DeepPartial<CompanyWorkspaceEntity>[]) {
     return companies.reduce(
       (acc, company) => {
-        if (!company.domainName) {
+        if (!company.domainName?.primaryLinkUrl || !company.id) {
```
Comment (FelixMalfait) [inline packages/twenty-server/src/modules/contact-creation-manager/services/create-company.service.ts:159]: curious why this came now? wouldn't this fail two line laters if domainName is null?

**Review by FelixMalfait — COMMENTED** [review]
Great! Let's do the same for emails and phones

**Pull request merged**